amundsen_application/static/js/ducks/rootSaga.ts

```diff
@@ -100,12 +103,13 @@ export default function* rootSaga() {
     // TableDetail
     getTableDataWatcher(),
     getColumnDescriptionWatcher(),
-    getLastIndexedWatcher(),
     getPreviewDataWatcher(),
     getTableDescriptionWatcher(),
     updateColumnDescriptionWatcher(),
     updateTableDescriptionWatcher(),
     updateTableOwnerWatcher(),
+    // LastIndexed
```

Comment:
Nit: I would say these comments are not necessary

Reply:
Done just for consistency purposes. I can remove them though. Should I remove all the comments in this file or just new ones added in this PR?

Reply:
I'd prefer to leave them in.  I believe these comments aren't necessary for me, but I put them in because since we happened to organize this list in order of section vs. alphabetically I figured the comments helps give some context.

Reply:
Okay, left them as it is.